plenum/server/node.py

```diff
@@ -992,7 +991,8 @@ def start(self, loop):
         self.logNodeInfo()
 
     def schedule_initial_propose_view_change(self):
-        self.lost_primary_at = time.perf_counter()
+        # It is supposed that master's primary is lost until it is connected
+        self.primaries_disconnection_times[self.master_replica.instId] = time.perf_counter()
```

Comment:
Should we do the same logic for backup replicas too?

Reply:
For a backup replica `Node.primary_selected` will schedule its removal in case its primary has been disconnected.